src/scss/components/alert/index.scss

```diff
@@ -1,35 +1,35 @@
 hx-alert {
-  background-color: $gray-400;
-  color: $gray-1000;
+  background-color: var(--hxAlert, $gray-400);
```

Comment:
@BY00565233, the CSS variables need to be more explicit to help us provide more context on usage.

So in this case, all CSS variables would be scoped to `hxAlert`.

```css
background-color: var(--hxAlert-bgcolor, $gray-400);
```

Reply:
Changes have been done,Please review